shared/actions/chat2/index.tsx

```diff
@@ -2353,7 +2362,32 @@ const markThreadAsRead = async (
   if (meta) {
     readMsgID = message ? (message.id > meta.maxMsgID ? message.id : meta.maxMsgID) : meta.maxMsgID
   }
-  logger.info(`marking read messages ${conversationIDKey} ${readMsgID}`)
+
+  logger.info(`marking read messages ${conversationIDKey} ${readMsgID} for ${action.type}`)
+  await RPCChatTypes.localMarkAsReadLocalRpcPromise({
+    conversationID: Types.keyToConversationID(conversationIDKey),
+    msgID: readMsgID,
+  })
+}
+
+const markAsUnread = async (
+  state: Container.TypedState,
+  action: Chat2Gen.MarkAsUnreadPayload,
+  listenerApi: Container.ListenerApi
+) => {
+  if (!state.config.loggedIn) {
+    logger.info('bail on not logged in')
+    return
+  }
+  const conversationIDKey = action.payload.conversationIDKey
```

Comment:
I know this is mostly a copy and paste but we do have a lot of old-ish code in the codebase with many older patterns so I think we should try and get newer code to follow some of the newer style (admittedly not clearly documented anywhere) so some of this feedback isn't required but mostly to document things and considered 🐑  / nice to have. In this case I think this style is a little nicer

```
const {conversationIDKey, readMsgID} = action.payload
```